lib/widgets/splash.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:async';
```

Comment:
This whole file is deleted now, not required.

Reply:
Ok